app/assets/stylesheets/application.css.scss

```diff
@@ -155,3 +155,23 @@ nav ul.nav.navbar-nav> li > a {
   padding-left: 8px;
   padding-right: 8px;
 }
+.right-bar a {
```

Comment:
could we use the bem style for markup ?

Reply:
I followed the code style adopted in the project